### DISCUSSION TITLE
module 'datapane' has no attribute 'PageLayout'

### DISCUSSION THREAD

**Discussion by ArtemSokolov** [datapane/datapane#364] [Ideas]
Dear Datapane developers,

It seems that the latest update broke the previous workflow for displaying pages with a sidebar, e.g.,
```
dp.App(
  dp.Page(page1, title="Page 1"),
  dp.Page(page2, title="Page 2"),
  layout=dp.PageLayout.SIDE
)
```
now produces `AttributeError: module 'datapane' has no attribute 'PageLayout'`

What is the suggested alternative for rendering side bars? For some reason, I can no longer find the corresponding example in the docs.

My apologies if I am missing something obvious.

-Artem

**Comment by shahinrostami** [comment]
Hello @ArtemSokolov! Thank you for raising this.

`PageLayout` is not available since the `0.16.0` update - apologies for the inconvenience.

We're going to be looking at this as part of our upcoming improvements. I've labelled this as an enhancement so we can keep track of it!

**Comment by mands** [comment]
#365 covers the same issue, upvoting here
